adapters/between/betweentest/params/race/banner.json

```diff
@@ -0,0 +1,5 @@
+{
+  "zone": "2eb6bd58-865c-47ce-af7f-a918108c3fd2",
```

Comment:
This file looks like a copy-paste leftover. The params here should be the params that you expect in the between bidder ext which for you is just `host`